.github/workflows/checks.yml

```diff
@@ -91,6 +91,8 @@ jobs:
       - uses: actions/checkout@v4
       - name: Set up environment
         uses: ./.github/actions/setup
+      - name: Install LCOV
+        run: sudo apt-get install lcov
```

Comment:
I'm afraid this is adding a dependency to the CI and we should be very careful about it. Is there any way to verify `lcov` is trusted?